pkgs/applications/networking/mailreaders/neomutt/default.nix

```diff
@@ -35,15 +35,19 @@
   withContrib ? true,
 }:
 
+assert lib.warnIf (
+  enableMixmaster
+) "Support for mixmaster has been removed from neomutt since the 20241002 release" true;
+
```

Comment:
Why not just drop the option entirely?

Reply:
I think it would be helpful to display a message to folks that use the `enableMixaster`, so they become aware of the change. After a few more releases the option could and probaly should be removed.